core/openapi/schemas/language.yaml

```diff
@@ -0,0 +1,34 @@
+type: object
+description:
+  The language used for textual values in this record.
+required:
+  - code
+properties:
+  code:
+    type: string
+    description:
+      The language tag as per RFC-5646.
+    example:
+      code: el
+    name:
+      type: string
```

Comment:
For name and alternate I've set `minLength: 1` in the STAC schema. Empty string don't seem overly useful.